### DISCUSSION TITLE
JK bms ble some information reversed.

### DISCUSSION THREAD

**Discussion by SilverMP** [Louisvdw/dbus-serialbattery#881] [Q&A]
Hello.

I hope i am in the right place.

First of all, wonderful driver, Thank you !

Installed driver, and works from day one with serial connection on cerbo gx

Some days back i installed jk bms with broken rs port and connected it via bluetooth.

Connected with no issues and went with ease but some info is reversed, and one missing.

I attached screenshots.
![main screen](https://github.com/Louisvdw/dbus-serialbattery/assets/151245014/1c36cd5c-15c9-4428-9f80-39619ffbd635)
SOC Reversed
![jk bms BLE 1](https://github.com/Louisvdw/dbus-serialbattery/assets/151245014/ec969426-31d6-4381-b6d0-e8c0f0de119e)
mosfet temperature missing
![jk bms ble 2](https://github.com/Louisvdw/dbus-serialbattery/assets/151245014/f52ee9e4-6afb-46d6-a97c-9cfb74b02bd7)
installed/available reversed

What did i mess up ? :)

**Comment by mr-manuel** [comment]
Next time would better to open a [support request](https://github.com/Louisvdw/dbus-serialbattery/issues/new?assignees=&labels=support&projects=&template=support_request.yml), since there we get all data we need.

Probably you are using driver version `v1.0.20230531`. Please install the latest nightly version from the dev branch, that should solve your issue with the MOSFET temperature.

What information is missing and which is reversed?

**Reply by SilverMP** [reply]
I am very sorry, i will keep that in mind and act as follows !

Will try nightly version at the weekend, at the moment not possible. ( 40km away ). and find out if mosfets temp come back.

SOC (state of charge) reversed, more i consume out, more the number increases (in other mind, its logigal, shows how much % is consumed hehe ).

Installed/available amphours reversed.
